### PR TITLE
=== Feature Freeze 2016 ===

### PR DESCRIPTION
**Pull request by Jordie0608**
winter-is-coming

### CODE REVIEW

**Comment by KorPhaeron** [comment]
This is an American codebase sorry

**Comment by swankcookie** [comment]
yeah, what country is that?

**Comment by Incoming5643** [comment]
We still got twelve whole hours before the freeze

**Comment by Jordie0608** [comment]
Aussie rules, same as last time

**Comment by Thunder12345** [comment]
It's still 2015, get out of here with your early freezes

**Comment by Incoming5643** [comment]
Australia's existence is purely theoretical

**Comment by Thunder12345** [comment]
Also you forgot to give it every single label like usually happens

**Comment by duncathan** [comment]
>conflicts

**Comment by bgobandit** [comment]
U
S
A

U
S
A

**Comment by Cheridan** [comment]
plz fix conflicts so we can merge

**Comment by Iamgoofball** [comment]
>still 2015

Yeah no I got shit I'm doing, freeze don't start till 2016

File: config/admins.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
-Optimumtact = Host
+Optimumtact = Nerdlord
```
Comment (Jalleo) [inline config/admins.txt]: Pfft

**Comment by Bawhoppen** [comment]
do i still have time

File: config/admins.txt [hunk 2]
```diff
@@ -103,4 +103,4 @@ SuperSayu = Game Master
 Lzimann = Game Master
 As334 = Game Master
 neersighted = Game Master
-Swankcookie = Game Master
\ No newline at end of file
+Swankcookie = Game Master
```
Comment (swankcookie) [inline config/admins.txt]: Did i SAY you could touch my shit, bruh??

**Comment by Shadowlight213** [comment]
I read the branch name as I bet there will still be an aussie from 2013

**Comment by Dorsisdwarf** [comment]
Of course there'll be an issue from 2013, we've got a Buggson in there and they're a pain because people don't like them closed or something

File: config/admins.txt (hunk 1 above)
Comment (tedward1337) [inline config/admins.txt]: holy kek

**Comment by tkdrg** [comment]
it's over

**Comment by PKPenguin321** [comment]
wait the freeze is over 100% now??

**Comment by RemieRichards** [comment]
@PKPenguin321 there was an end early condition (a "thaw"), and we hit it.

**Comment by PKPenguin321** [comment]
YYYYYYYYYYEAAAAAAAAAAAHHHHHHHHHHHHHHHHHHHHHHHH

**Pull request closed without merging**